headers/types/common/common.h

```diff
@@ -897,6 +897,34 @@ struct wan_table {
 };
 ASSERT_SIZE(struct wan_table, 5392);
 
+struct sound_envelope_parameters {
+    uint8_t use_envelope;
+    uint8_t slide_time_multiplier;
+    uint16_t field2_0x2;
+    uint32_t field3_0x4;
+    uint8_t attack_begin;
+    uint8_t attack_time;
+    uint8_t decay_time;
+    uint8_t sustain_level;
+    uint8_t hold_time;
+    uint8_t sustain_time;
+    uint8_t release_time;
+    uint8_t field11_0xe;
+};
+ASSERT_SIZE(struct sound_envelope_parameters, 16);
+
+struct sound_envelope {
+    struct sound_envelope_parameters parameters;
+    int32_t current_volume;
+    int32_t volume_delta;
+    int32_t ticks_left;
```

Comment:
`int` is more commonly seen throughout the repo, although I'm not sure if it's required since `int32_t` is also used sometimes.

Reply:
Yeah either is fine, I don't think it matters much.

Reply:
Alright. I personally prefer `int` since it's simpler, but I guess either works.

Reply:
I'll go ahead and use the more prevalent `int`.